osu!idol 2019 audition announcement (#2325)
ppy/osu-wiki: news/2019-07-02-osu-idol-2019-auditions-now-open.md

---
layout: post
title: "osu!idol 2019 Auditions Now Open"
date: 2019-07-02 12:00:00 +0000
---

> So do you enjoy singing? Are you interested in competing to become the next osu!idol? Plug in your mic and come in, it’s that time again to join and show off those singing skills!

Reviewer: ```suggestion
So do you enjoy singing? Are you interested in competing to become the next osu!idol? Plug in your mic and come in, it's that time again to join and show off those singing skills!
```
`’` -> `'`

![](/wiki/shared/news/2019-07-02-osu-idol-2019-auditions-now-open/banner.jpg)

### Can you tell me what this is?

osu!idol is the community driven singing contest! Contestants participate solo or as a duet in multiple stages consisting of various genres, showcasing what they got to our panel of judges. They will evaluate the performances and determine who makes it through the stages, eventually ending up in the finals. Voting for the winner will be conducted by you, the community, to decide who will be the next osu!idol!

### So what do I do?

Record a clip around 30 seconds long of your best singing with no background music added. Submissions will be lasting all month long, so give us the best you got! 

> ### What’s in it for me?

Reviewer: ```suggestion
### What's in it for me?
```

Once community voting has concluded, the finalists will be receiving these prizes as followed:

* **1st:** osu!idol 2019 winner profile badge + 6 months supporter tag
* **2nd:** 3 months supporter tag
* **3rd:** 1 month supporter tag

### Where do I sign up?

Check out the [**official thread**](https://osu.ppy.sh/illdoitsoon) for the contest rules, submission sheet, and more details and discussions on the event. Make sure to join the [discord server](https://discord.gg/7hZgNeH) as well to get announcements and updates, ask questions, and talk to the community!

-Kyonko Hizara

> 

Reviewer: ```suggestion
```

(remove this trailing space)
